ambry-api/src/main/java/com.github.ambry/messageformat/BlobProperties.java

```diff
@@ -32,15 +32,15 @@
   protected long creationTimeInMs;
 
   /**
-   * @param blobSize The size of the blob in bytes
+   * @param blobSize The size of the blob in bytes.
    * @param serviceId The service id that is creating this blob
```

Comment:
Minor: could periods either be added to the other javadoc lines or these periods be removed.